test/e2e/specs/editor/blocks/columns.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('Columns', () => {
+	test.beforeEach(async ({ admin }) => {
+		await admin.createNewPost();
+	});
+
+	test.afterEach(async ({ requestUtils }) => {
+		await requestUtils.deleteAllPosts();
+	});
+
+	test('restricts all blocks inside the columns block', async ({
+		page,
+		editor,
+	}) => {
+		// Open Columns
+		await editor.insertBlock({ name: 'core/columns' });
+		await page.locator('[aria-label="Two columns; equal split"]').click();
+
+		// Open List view toggle
+		await page.locator('role=button[name="List View"i]').click();
+
+		// block column add
+		await page
+			.locator(
+				'[aria-label="Block navigation structure"] >> text=Column'
+			)
+			.nth(1)
```

Comment:
```suggestion
			.first()
```

Suggestion (non-blocking): You can also use the `first()` short-form here.